lib/make-disk-image.nix

```diff
@@ -94,7 +94,7 @@ let
 
     # We copy files with cp because `nix copy` seems to have a large memory leak
     mkdir -p ${systemToInstall.config.disko.rootMountPoint}/nix/store
-    xargs cp --recursive --target ${systemToInstall.config.disko.rootMountPoint}/nix/store < ${closureInfo}/store-paths
+    xargs -P 0 -I {} sh -c 'cp --recursive --target ${systemToInstall.config.disko.rootMountPoint}/nix/store "$1" || exit 255' sh {} < ${closureInfo}/store-paths
```

Comment:
Does this not run than one cp for every store path?

Reply:
Currently yes. We could wrap the cp invocation in a short script with `$@` to avoid that.

Reply:
Addressed by https://github.com/nix-community/disko/pull/910/commits/e6c2ca8c24a1a30ef24e5856436360a1db84a197